src/components/navbar/HomeNav.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import { routes } from '../../constants/routesPaths';
+import smilies from '../../assets/smilies.svg';
+import avatar_example from '../../assets/avatar-example.svg';
```

Comment:
Always use camel case in javascript. I mean `avatarExample` instead of `avatar_example`